source/08-sushi/02-authentication-and-security-for-counter-sushi-api.rst

```diff
@@ -16,3 +16,5 @@ The API MUST be secured using one or more of the following methods:
 Non-standard techniques for authentication (techniques not specified in the COUNTER_SUSHI API specification) MUST NOT be used.
 
 If IP address authentication is implemented, it MUST allow the same SUSHI client (a single IP address) to harvest usage for multiple customer accounts (e.g. hosted ERM services).
+
+If global reports are available, the Customer_ID 0000000000000000 MUST be used for "The World". This follows the 16-digit ISNI structure but is an empty record. 
```

Comment:
The MUST seems wrong to me since global reporting is optional. Rules related to "The World" so far are "should" (in lower case) rules.

This rule is not security related, in my opinion it should instead be included in the Institution_ID description in Table 3.f, since this value actually would have to be returned as Institution_ID (with type ISNI?).

Reply:
If we want to address the authentication question, we also would have to recommend how this customer ID should be used in combination with a requestor ID or API key. Could any valid requestor ID and/or API key be used in combination with this customer ID? Or would a specific requestor ID or API key be required?

Note that it must be customer_id instead of Customer_ID. Since we remove the JSON Customer_ID header, Customer_ID in 5.1 only refers to the common extension.

It's still not clear to me whether the value would be returned in the Institution_ID header as ISNI:0000000000000000. even though this isn't a valid ISNI, or as a proprietary ID? The best place to clarify that probably would be Sections 4.1.1, 4.2.1, 4.3.1 and 4.4.1.

Reply:
I don't think this should go into section 4 but have amended section 8